### PR TITLE
Enhance version check to differentiate between ESP32 and ESP32-S3 bin versions #1240

### PR DESCRIPTION
**Pull request by proddy**
Haven't tested on real ESP32 variants yet

### CODE REVIEW

**Comment by MichaelDvP** [comment]
Tested on esp32S3 and esp32, working.
For the changelog it is maybe better to redirect to the download page, instead of downloading the md-file. I think most people does not have a reader/editor associated with *.md and don't know how to open.

**Comment by proddy** [comment]
> Tested on esp32S3 and esp32, working. For the changelog it is maybe better to redirect to the download page, instead of downloading the md-file. I think most people does not have a reader/editor associated with *.md and don't know how to open.

thanks for testing.

I couldn't find a way to show the md file in the browser (using the raw URL or installing another plugin). Any ideas here or shall we just leave it?

**Comment by MichaelDvP** [comment]
We can leave it, but i think https://github.com/emsesp/EMS-ESP32/blob/dev/CHANGELOG_LATEST.md shows the formated dev changelog and https://github.com/emsesp/EMS-ESP32/blob/main/CHANGELOG.md the release. But it's not the file from the release asset and may be different (normally not).

File: platformio.ini [hunk 1]
```diff
@@ -75,8 +74,8 @@ build_unflags = ${common.unbuild_flags}
 
 ; for github Actions actually standard platform is buggy, use dev-platform
 [env:ci_s3]
+extends = espressi32_base
 platform = https://github.com/platformio/platform-espressif32.git
```
Comment (MichaelDvP) [inline platformio.ini:78]: You need to remove this line 78, it overwrites the platform from the base.

**Comment by proddy** [comment]
merged by accident, I still need to put in direct links to the md files

**Pull request merged**